scripts/prepare_sector_network.py

```diff
@@ -851,19 +851,17 @@ def annuity_factor(v):
     return costs
 
 
-def add_generation(n, costs):
+def add_generation(n, costs, existing_capacities, existing_generation):
```

Comment:
Here shouldn't be existing_efficiencies as a second input?

Reply:
Ah yes! My bad... I'll change that